README.md

```diff
@@ -1 +1,158 @@
-# SetUp-With-Webpack
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠️ Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To Do List Rod] 
+
+This is an application that has the function of being a reminder of pending tasks.
+
+## 🛠️ Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://html.org/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://css.com/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features 
+
+- *[Add HTML]*
+- *[Add CSS]*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo
+
+To see the display of my portfolio use this link: (https://github.com/rodrigopenayo98/ToDo-List-Rod)

```

Comment:
- [ ] This link is leading to your project repository, please add a correct link to your live version if the project requirements asked for it or remove it if they don't